pkg/apis/postgres-operator.crunchydata.com/v1beta1/patroni_types.go

```diff
@@ -48,6 +52,21 @@ type PatroniSpec struct {
 	// - https://patroni.readthedocs.io/en/latest/kubernetes.html
 }
 
+type PatroniLogConfig struct {
+
+	// Patroni Log file size
+	// https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity
+	// +required
+	FileSize *string `json:"fileSize"`
```

Comment:
I want the field we expose to restrict the _total_ space occupied by all log files combined. I think I prefer `limit` or `storageLimit` for this. WDYT?

----

I went looking for some examples in the Kubernetes API:

CSIStorageCapacity: `maximumVolumeSize`
PodDisruptionBudget: `maxUnavailable`, `minAvailable` (Pods)
LimitRange: `max: { cpu, memory }`, `maxLimitRequestRatio: { cpu, memory }`
QueuingConfiguration: `queueLengthLimit`
TopologySpreadConstraint: `maxSkew`, `minDomains`
HorizontalPodAutoscaler: `maxReplicas`, `minReplicas`

Typing things out:

```yaml
history: 10Mi
maximumHistory: 10Mi
limit: 10Mi
storageLimit: 10Mi

# Knowing the Patroni YAML, this is ambiguous to me.
#  - Is this a single file, or is this all files combined?
maximumSize: 10Mi
maximumTotalSize: 10Mi
```

Reply:
`storageLimit` makes sense to me, especially in light of the [discussion above](https://github.com/CrunchyData/postgres-operator/pull/4047#discussion_r1888951453). I'll make that change.